pkg/integration/integration.go

```diff
@@ -87,7 +87,7 @@ func (*Integration) Activate(name string, namespace string, activeFilters []stri
 
 	if !skipInstall {
 		if err := integrations[name].Deploy(namespace); err != nil {
-			return err
```

Comment:
This hides the inner error

Reply:
it indeed wraps the error, this looks good to me,

but I am not sure about the value of these integration tests, they seem a bit pedantic I think